lib/loader/index.js

```diff
@@ -83,51 +87,39 @@ function preInstantiate(imports) {
   return extendedExports;
 }
 
-const E_NOEXPORTRUNTIME = "Operation requires compiling with --exportRuntime";
-const F_NOEXPORTRUNTIME = function() { throw Error(E_NOEXPORTRUNTIME); };
-
 /** Prepares the final module once instantiation is complete. */
 function postInstantiate(extendedExports, instance) {
   const exports = instance.exports;
   const memory = exports.memory;
   const table = exports.table;
-  const __new = exports.__new || F_NOEXPORTRUNTIME;
-  const __pin = exports.__pin || F_NOEXPORTRUNTIME;
-  const __unpin = exports.__unpin || F_NOEXPORTRUNTIME;
-  const __collect = exports.__collect || F_NOEXPORTRUNTIME;
+  const __new = exports.__new || F_NO_EXPORT_RUNTIME;
+  const __pin = exports.__pin || F_NO_EXPORT_RUNTIME;
+  const __unpin = exports.__unpin || F_NO_EXPORT_RUNTIME;
+  const __collect = exports.__collect || F_NO_EXPORT_RUNTIME;
   const __rtti_base = exports.__rtti_base;
-  const getRttiCount = __rtti_base
+  const getRttCount = __rtti_base
```

Comment:
Why drop the `i`?

Reply:
For simplicity and for match with naming in Wasm GC

Reply:
Hmm, in Wasm GC it stands for "runtime type", though, while the loader accesses not types but "runtime type info", conceptionally.